components/Nav.js

```diff
@@ -298,6 +363,12 @@ export default class Header extends Component {
     document.body.classList.toggle('menu-opened');
```

Comment:
**Do not reference the DOM directly through react!!!!** You must go through state:
eg `<body className={this.state.menuOpened ? 'menu-opened' : ''}>`